Umbraco-Cloud/Set-Up/Manage-Subscriptions/index.md

```diff
@@ -0,0 +1,52 @@
+---
+versionFrom: 8.0.0
+---
+
+# Manage Subscriptions
+
+In this article you will be able to find information on how to manage your subscriptions. You will also be able to find information on how to change ownership of your projects.
+
+## Invoices
+You are able to find all you invoices when you log into [the shop](https://shop.umbraco.com/profile/sign-in?returnURL=%2fprofile).
+
+Once you have logged in you can see an overview of your account where you will be able to find your [Order History](https://shop.umbraco.com/profile/options/order-history/). It is in this section you are able to see all your invoices.
+
+![Overview of your account](images/account-overview.png)
+
+Under the Order History you are able to get an overview of all your invoices. The overview has the following details.
+- Order number
+- Total price ex. VAT
+- Date of purchase
+- Payment method
+- Status of the order
+- A download link for the invoice
+
+![Order History](images/order-history.png)
+
+:::note
+If you have not yet paid for your invoice, the download link will change to a payment link.
+:::
+
+## Changing your Credit Card information
+
+If you have to change your credit card information you can do that under [Manage Subscriptions](https://shop.umbraco.com/profile/options/manage-subscriptions/). When changing your credit card information you will have to change the information on all subscriptions where the credit card is being used.
+
+## How to change ownership of subscriptions and projects
+
+You might find yourself if a situation where you will need to change the ownerships of subscriptions or projects. That can be done in the following ways.
+
+### Subscriptions
```

Comment:
I think we need something in this section about how the one taking over the subscription needs to have an account on the shop, in order for us to assign the ownership of the subscription to them.
What do you think? 🤔